language-support/ts/daml-ledger/index.ts

```diff
@@ -873,6 +894,58 @@ class Ledger {
     }
     return this.streamSubmit("streamFetchByKeys", template, 'v1/stream/fetch', request, reconnectRequest, initState, change);
   }
+
+  /**
+   * Fetch parties by identifier.
+   *
+   * @param parties An array of Party identifiers.
+   *
+   * @returns An array of the same length, where each element corresponds to
+   * the same-index element of the given parties, ans is either a PartyInfo
+   * object if the party exists or null if it does not.
+   *
+   */
+  async fetchPartiesInfo(parties: Party[]): Promise<(PartyInfo | null)[]> {
```

Comment:
The underlying ledger API calls this `getParties` and the one below `listKnownParties`. Should we try to match those names? Or maybe at least remove the `Info` part to (roughly) match the headers in the JSON API docs? Not quite sure what the best approach here is so very much open to suggestions or to just keeping it as at is. I guess overloading and using only a single name might also work.

Reply:
Matching ledger API names seems like a good idea. I should have checked.

Reply:
Thanks! lgtm now